docs/manual/form-generator/forms.en.md

```diff
@@ -6,13 +6,10 @@ aliases:
 weight: 10
```

Comment:
title: "Forms"
description: "With the form generator you can create forms and either send their data by e-mail or write them into the database."